aws/resource_aws_lightsail_instance_test.go

```diff
@@ -187,6 +199,28 @@ resource "aws_lightsail_instance" "lightsail_instance_test" {
   availability_zone = "us-east-1b"
   blueprint_id      = "gitlab_8_12_6"
   bundle_id         = "nano_1_0"
+  tags = {
+    Name = "tf-test"
+  }
+}
+`, lightsailName)
+}
+
+func testAccAWSLightsailInstanceConfig_updated(lightsailName string) string {
+	return fmt.Sprintf(`
+provider "aws" {
```

Comment:
Please note that we are trying to remove hardcoded `provider` configurations from the test configurations and there is a pending PR for fixing Lightsail here: https://github.com/terraform-providers/terraform-provider-aws/pull/9080

Reply:
I think I've done it correctly this time but please check.